SCF/Assignment1/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
never commit .classpath because it is workspace setting and everyone have different workspace setting.